features/stream/src/main/java/com/m3u/features/stream/StreamMask.kt

```diff
@@ -493,11 +564,36 @@ internal fun
                                     }
                             )
                         }
-
                     }
                 }
             },
-            modifier = modifier
         )
     }
 }
+
+
+@Composable
+private fun Programme.readText(): AnnotatedString {
+    val preferences = hiltPreferences()
+    val clockMode = preferences.twelveHourClock
+    val title = title
+
+    return buildAnnotatedString {
+        val start = Instant.fromEpochMilliseconds(start)
+            .toLocalDateTime(TimeZone.currentSystemDefault())
+            .formatEOrSh(clockMode)
+
+
+        if (isNew) {
+            val indexOfSubstring = title.indexOf("ᴺᵉʷ")
```

Comment:
bad code

Reply:
whats wrong? im using it to remove this line where i have the custom text icon because it doesnt need this information on both sides of the title. i did not want to edit it in the database as you suggested not to. you can choose to not include this code if you like

Reply:
save the origin whatever it is.

Reply:
Your code will always drop the word after the "new". If some title start with it, the result will be a blank.